sentences/sv/scene_HassTurnOn,yaml

```diff
@@ -0,0 +1,27 @@
+language: sv
+intents:
+  HassTurnOn:
+    data:
+      - sentences:
+          - "[aktivera] <name> [scene]"
+          - "<name> on"
```

Comment:
Is "<name> on" used as:
"Kvällsmode on" in that case I would probably use "Kvällsmode till"